sdcm/cluster_docker.py

```diff
@@ -78,6 +78,19 @@ def __init__(self,  # pylint: disable=too-many-arguments
             assert int(container.labels["NodeIndex"]) == node_index, "Container labeled with wrong index."
             self._containers["node"] = container
 
+    @property
+    def network_interfaces(self):
+        pass
+        # TODO: it did not work, we can it check later
```

Comment:
Is there anything we need to do for docker backend ?

Reply:
It did not work me. I did not investigated yet